src/Redux/Reducers-Redux/authSlice.js

```diff
@@ -1,91 +1,94 @@
 import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
 import axios from "axios";
-import {notifySuccess,notifyError} from "Utilities/Notifications";
+import { notifySuccess, notifyError } from "Utilities/Notifications";
 
-const initialState={
-    currentUser:{},
-    loading:false
-    }
+const initialState = {
+  currentUser: {},
+  loading: false,
+};
 
-export const login=createAsyncThunk("auth/login",async(userDetails)=>{
-try {
-    const response=await axios.post(`/api/auth/login`,{
-    username:userDetails.username,
-    password:userDetails.password,
+export const login = createAsyncThunk("auth/login", async (userDetails) => {
+  try {
+    const response = await axios.post(`/api/auth/login`, {
+      username: userDetails.username,
+      password: userDetails.password,
     });
     return response.data;
-} catch (error) {
-  console.log(error.response.data.errors[0]);  
-}
+  } catch (error) {
+    console.log(error.response.data.errors[0]);
+  }
 });
-export const signUp=createAsyncThunk("auth/signUp",async(userDetails)=>{
+export const signUp = createAsyncThunk("auth/signUp", async (userDetails) => {
+  try {
+    const response = await axios.post(`/api/auth/signup`, {
+      username: userDetails.username,
+      password: userDetails.password,
+      name: userDetails.name,
+    });
+    return response.data;
+  } catch (error) {
+    console.log(error.response);
+  }
+});
+
+export const checkToken = createAsyncThunk("auth/checkToken", async () => {
+  const encodedToken = localStorage.getItem("token");
+  if (encodedToken) {
     try {
-        const response= await axios.post(`/api/auth/signup`,{
-            username:userDetails.username,
-            password:userDetails.password,
-            name:userDetails.name
-        });
-        return response.data
+      const response = await axios.post("api/auth/verify", {
+        encodedToken: encodedToken,
+      });
+      console.log(response);
```

Comment:
you can remove console.log if not required

Reply:
yeah you are right
Thanks for reviewing and noticing it